packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -1,34 +1,24 @@
 /** @jsx withSlots */
 import * as React from 'react';
-import { Platform, Pressable, View } from 'react-native';
+import { Pressable, Platform } from 'react-native';
 import { checkboxName, CheckboxType, CheckboxProps } from './Checkbox.types';
 import { TextV1 as Text } from '@fluentui-react-native/text';
 import { stylingSettings, getDefaultSize } from './Checkbox.styling';
 import { compose, mergeProps, withSlots, UseSlots } from '@fluentui-react-native/framework';
 import { useCheckbox } from './useCheckbox';
 import { Svg, Path } from 'react-native-svg';
 
-const unsupportedAndroidProps: string[] = ['size', 'shape', 'required', 'labelPosition', 'tooltip'];
-
 export const Checkbox = compose<CheckboxType>({
   displayName: checkboxName,
   ...stylingSettings,
   slots: {
     root: Pressable,
-    checkbox: View,
+    checkbox: Pressable,
     checkmark: Svg,
     label: Text,
     required: Text,
   },
   useRender: (userProps: CheckboxProps, useSlots: UseSlots<CheckboxType>) => {
-    // Removal of tokens which are not supported for Android
-    {
-      Platform.OS === 'android' &&
-        unsupportedAndroidProps.map((e) => {
-          Reflect.deleteProperty(userProps, e);
-        });
-    }
-
```

Comment:
@Saadnajmi based on internal discussions, we have decided not to remove the props, if a user passes it, it will render it, but we will not support it which is not present on particular platform spec. 

We will go with Tester App where we will show examples of what is supported for particular platform.